app/next.config.mjs

```diff
@@ -38,15 +42,16 @@ const config = {
       },
     ];
   },
+  async headers() {
+    return [
+      {
+        source: "/:path*",
+        headers: securityHeaders,
+      },
+    ];
+  },
 };
 
-let withBundleAnalyzer;
-if (process.env.ANALYZE === "true") {
-  withBundleAnalyzer = require("@next/bundle-analyzer")({
-    enabled: process.env.ANALYZE === "true",
-  });
-} else {
-  withBundleAnalyzer = (config) => config;
-}
-
-module.exports = withBundleAnalyzer(config);
+export default withBundleAnalyzer({ enabled: process.env.ANALYZE === "true" })(
+  nextConfig
+);
```

Comment:
@Atmosfearful - ran into a `SyntaxError: Invalid Statement` error trying to keep the previous implementations intention.  I don't think it's too big of a deal to export with the bundle analyser and rely on passing a flag to enable the analyser.

I'll take another stab at it tomr if you're not happy with it, need to sleep now for my meatspace job tomr 😴